doc/commands/history.8.rst

```diff
@@ -65,16 +65,56 @@ Subcommands
     | Replay the transaction that was previously stored into the file.
 
 
-Options
-=======
+Options for ``list`` and ``info``
+=================================
 
 ``--reverse``
     | Reverse the order of transactions in the output.
 
 
+Options for ``undo``
+====================
+
+``--skip-unavailable``
+    | Allow skipping packages actions that are not possible perform.
+
+``--ignore-extras``
+    | Don't consider extra packages pulled into the transaction as errors.
+    | They will still be reported as warnings.
+
+``--ignore-installed``
+    | Don't consider mismatches between installed and stored transaction packages as errors.
+    | For install actions skip already installed packages.
+    | For upgrade actions skip groups or environments that are not installed.
+    | For remove actions skip not installed packages/groups/environments.
+    | They will still be reported as warnings.
+    | Using this option result in an empty transaction.
```

Comment:
Maybe move the last two lines right after the first `Don't ...` line? Otherwise it's not really clear what `They will still be reported as warnings.` is referring to.

Reply:
Sure, moved.